src/Analyzers/CSharp/Tests/RemoveUnnecessaryNullableDirective/CSharpRemoveRedundantNullableDirectiveTests.cs

```diff
@@ -182,13 +182,41 @@ class Program
                 """
                 // File Header
 
-
                 class Program
                 {
                 }
                 """);
         }
 
+        [Fact]
+        public async Task TestRedundantDirectiveBetweenUsingAndNamespace()
+        {
+            await VerifyCodeFixAsync(
+                NullableContextOptions.Enable,
+                """
+                using System;
+                
+                [|#nullable enable|]
+
+                namespace MyNamespace
```

Comment:
💡 Also would be good to have tests with the following. Each of these two cases should result in a single blank line appearing after the refactoring.

```
using System;
#nullable enable

namespace MyNamespace
{
}
```

```
using System;

#nullable enable
namespace MyNamespace
{
}
```

Reply:
Done, had to handle the second one as I wasn't doing that